llvm/lib/Target/AIE/AIESlotCounts.cpp

```diff
@@ -0,0 +1,72 @@
+//===- AIESlotCounts.cpp - SlotCount utility ------------------------------===//
```

Comment:
nit: this looks like something for the `Utils` folder.

Reply:
Except that it rests on AIE facilities.